tests/libvmi/OWNERS

```diff
@@ -1,3 +1,12 @@
 # See the OWNERS docs at https://go.k8s.io/owners
-approvers:
-  - EdDev
+filters:
```

Comment:
I do not think this level of accuracy is needed.
The topic here is the VM/VMI spec, not network specific logic.

If we will find this is a problem, it can be added later.

Reply:
removed the commit.